tests/tests/tier0/bluechi-reset-failed/test_bluechi_reset_failed.py

```diff
@@ -0,0 +1,62 @@
+#
+# Copyright Contributors to the Eclipse BlueChi project
+#
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+import logging
+from typing import Dict
+
+from bluechi_test.config import BluechiAgentConfig, BluechiControllerConfig
+from bluechi_test.machine import BluechiAgentMachine, BluechiControllerMachine
+from bluechi_test.service import Option, Section, SimpleRemainingService
+from bluechi_test.test import BluechiTest
+
+LOGGER = logging.getLogger(__name__)
+NODE_FOO = "node-foo"
+
+
+def exec(ctrl: BluechiControllerMachine, nodes: Dict[str, BluechiAgentMachine]):
+
+    node_foo = nodes[NODE_FOO]
+
+    # Create a service which won't start due to nonexistent executable
+    failed_service_1 = SimpleRemainingService(name="simple1.service")
+    failed_service_1.set_option(Section.Service, Option.ExecStart, "/s")
+    failed_service_2 = SimpleRemainingService(name="simple2.service")
+    failed_service_2.set_option(Section.Service, Option.ExecStart, "/s")
+    failed_service_3 = SimpleRemainingService(name="simple3.service")
+    failed_service_3.set_option(Section.Service, Option.ExecStart, "/bin/sleep 60")
```

Comment:
`failed_service_3` doesn't fail. So renaming the variable to `sleeping_service` or `succeeding_service` is clearer.